create-schnack/index.js

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env node
+/* eslint no-console: "off" */
+const fs = require('fs');
+const path = require('path');
+const { spawn } = require('child_process');
+const findUp = require('find-up');
```

Comment:
This is not used.

Can be removed from `package.json` too.

Reply:
thx